lib/types.js

```diff
@@ -63,55 +63,26 @@ module.exports = function (expect) {
         });
     }
 
-    // If Symbol support is not detected, default to passing undefined to
-    // Array.prototype.sort, which means "natural" (asciibetical) sort.
-    var keyComparator;
-    if (typeof Symbol === 'function') {
-        // Comparator that puts symbols last:
-        keyComparator = function (a, b) {
-            var aIsSymbol, bIsSymbol;
-            var aString = a;
-            var bString = b;
-            aIsSymbol = typeof a === 'symbol';
-            bIsSymbol = typeof b === 'symbol';
-            if (aIsSymbol) {
-                if (bIsSymbol) {
-                    aString = a.toString();
-                    bString = b.toString();
-                } else {
-                    return 1;
-                }
-            } else if (bIsSymbol) {
-                return -1;
-            }
-
-            if (aString < bString) {
-                return -1;
-            } else if (aString > bString) {
-                return 1;
-            }
-
-            return 0;
-        };
-    }
-
     expect.addType({
         name: 'object',
         indent: true,
         forceMultipleLines: false,
         identify: function (obj) {
             return obj && typeof obj === 'object';
         },
+        propertyStyle: 'property',
```

Comment:
Hmm, this is a pretty neat idea! It might be a bit more flexible and consistent with the `prefix` and `suffix` methods to introduce a `property` method in the type, though? Something like:

```js
property: function (output, key, inspectedValue) {
    return output.property(key, inspectedValue);
}
```